src/main/scala/Column.scala

```diff
@@ -1164,6 +1275,7 @@ case class Column private (
             errors =
               errors ++
                 validateLength(s.toString) ++
+                validateValue(s.toString) ++
```

Comment:
I thought we wanted to leave the datatype as-is when we parsed it? Why are we converting it to string here?

Reply:
Yeah, I think I forgot to make this change. Initially it was string, but then we decided to pass exact thing we got when parsed into validateValue function. I have changed it now.